PinpointKit/PinpointKit/Sources/MailSender.swift

```diff
@@ -96,24 +96,24 @@ public class MailSender: NSObject, Sender {
 private extension MFMailComposeViewController {
     
     func attach(_ feedback: Feedback) throws {
-        setToRecipients(feedback.recipients)
+        setToRecipients(feedback.configuration?.recipients)
         
-        if let subject = feedback.title {
+        if let subject = feedback.configuration?.title {
             setSubject(subject)
         }
         
-        if let body = feedback.body {
+        if let body = feedback.configuration?.body {
            setMessageBody(body, isHTML: false)
         }
         
-        try attach(feedback.screenshot, screenshotFileName: feedback.screenshotFileName)
+        try attach(feedback.screenshot, screenshotFileName: feedback.configuration?.screenshotFileName ?? "Screenshot")
         
         if let logs = feedback.logs {
-            try attach(logs, logsFileName: feedback.logsFileName)
+            try attach(logs, logsFileName: feedback.configuration?.logsFileName ?? "logs")
```

Comment:
This comes from the `develop` merge, originating in this PR: https://github.com/Lickability/PinpointKit/pull/138. `logsFileName` is now an optional property on `Feedback`’s `FeedbackConfiguration`, though it looks like it may not need to be optional. Since this originates with something on `develop`, I’m going to open a separate issue for it to be fixed there, and then later merged into `swift-2.3` and `swift-3.0`.